### PR TITLE
add note + fixes

### PR DESCRIPTION
**Pull request by mairaw**
Ran spellchecker and Acrolinx in a few of the articles

### CODE REVIEW

File: docs/architecture/cloud-native/application-resiliency-patterns.md [hunk 1]
```diff
@@ -5,37 +5,39 @@ ms.date: 06/30/2019
...
+As recommended in Chapter 1, Microsoft developers constructing cloud-native applications should be targeting .NET Core. Version 2.1 introduced the [HTTPClientFactory](https://www.stevejgordon.co.uk/introduction-to-httpclientfactory-aspnetcore) library for creating HTTP Client instances for interacting with URL-based resources. Superseding the original HTTPClient class, the factory class supports many enhanced features, one of which is [tight integration](../microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md) with the Polly resiliency library. With it, you can easily define resiliency policies in the application Startup class to handle partial failures and connectivity issues.
```
Comment (rpetrusha) [inline docs/architecture/cloud-native/application-resiliency-patterns.md]:
```suggestion
As recommended in Chapter 1, Microsoft developers constructing cloud-native applications should target .NET Core. Version 2.1 introduced the [HTTPClientFactory](https://www.stevejgordon.co.uk/introduction-to-httpclientfactory-aspnetcore) library for creating HTTP Client instances for interacting with URL-based resources. Superseding the original HTTPClient class, the factory class supports many enhanced features, one of which is [tight integration](../microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md) with the Polly resiliency library. With it, you can easily define resiliency policies in the application Startup class to handle partial failures and connectivity issues.
```

File: docs/architecture/cloud-native/azure-data-storage.md [hunk 2]
```diff
@@ -31,23 +33,23 @@ Note the alternatives in the previous figure when deploying a SQL Database:
...
+Azure SQL Database is a fully managed [Platform as a Service (PaaS) Database Engine](https://docs.microsoft.com/azure/sql-database/sql-database-paas) that handles  upgrading, patching, backups, and monitoring without user involvement. It always runs the latest stable version of the SQL Server Database Engine and patched OS and guarantees 99.99% availability. One feature, [active geo-replication](https://docs.microsoft.com/azure/sql-database/sql-database-active-geo-replication), lets you create readable secondary databases in the same or a different Azure data center. Upon failure, a failover to a secondary database can be initiated. At that point, the other secondaries automatically link to the new primary. Up to four secondary replicas are supported in either the same or in different regions, and these secondaries can also be used for read-only access queries.
```
Comment (rpetrusha) [inline docs/architecture/cloud-native/azure-data-storage.md]:
```suggestion
Azure SQL Database is a fully managed [Platform as a Service (PaaS) Database Engine](https://docs.microsoft.com/azure/sql-database/sql-database-paas) that handles upgrading, patching, backups, and monitoring without user involvement. It always runs the latest stable version of the SQL Server Database Engine and patched OS and guarantees 99.99% availability. One feature, [active geo-replication](https://docs.microsoft.com/azure/sql-database/sql-database-active-geo-replication), lets you create readable secondary databases in the same or a different Azure data center. Upon failure, a failover to a secondary database can be initiated. At that point, the other secondaries automatically link to the new primary. Up to four secondary replicas are supported in either the same or in different regions, and these secondaries can also be used for read-only access queries.
```

**Review by rpetrusha — APPROVED** [review]
LGTM. I've left two minor suggestions, @mairaw.

**Review by nishanil — APPROVED** [review]
Thank You @mairaw All the changes look great!

**Pull request merged**